content/news/playtest-20161001.md

```diff
@@ -0,0 +1,33 @@
+We are pleased to release the second playtest of this development cycle! This refines and fixes the many new features that we added in [playtest-20160904](/news/playtest-20160904/) in preparation for a final release.
+
+Over 5,000 people have tried out the previous playtest, which is about twice as many as usual. Thanks to everyone one of those people for testing and contributing back! Stable releases would not be possible without you. We now count on you to check out this playtest as rigorously as the first one. With any luck, this might be the last chance to report issues before the stable release, so don't miss it!
+
+As usual we have fixed quite a few bugs, many of them very visible and annoying. These include:
+
+* Servers becoming unusable in various ways (grey ping, kicking players on joining, etc.).
+* Some hotkeys not working (closing the ingame chat window, navigation hotkeys in the TD mod).
+* Aircraft reloading behaviour still having problems.
+* Support power speech notifications missing.
+* Crashes for various reasons.
+* Issues in mission scripts.
+
+Additionally, we have tweaked the balance of the RA mod some more. Most notably, spies have received the ability to infiltrate and reveal fake buildings, and the Soviet missile submarine now has an AA capability, so be sure to give those a good workout.
```

Comment:
I suggest breaking "Be sure to give those a good workout!" out into its own sentence.  It runs on a bit too long otherwise.